config.lua

```diff
@@ -398,24 +398,37 @@ barOptions:Initialize(function(self)
     title:SetPoint('TOPLEFT', 190, -10);
     title:SetText('Improved Blizzard UI - v'..GetAddOnMetadata('ImprovedBlizzardUI', 'Version'));
 
-    -- Cast Bar
+    -- Cast Bars
     local castBarTitle = self:CreateTitle();
     castBarTitle:SetPoint('TOPLEFT', 10, -50);
-    castBarTitle:SetText(Loc['Cast Bar']);
+    castBarTitle:SetText(Loc['Cast Bars']);
 
+    -- Player Cast Bar Timer
     local barTimer = self:CreateCheckButton('barTimer');
     barTimer:SetPoint('TOPLEFT', castBarTitle, 'BOTTOMLEFT', 0, -8)
-    barTimer:SetText(Loc['Casting Bar Timer']);
-    AddTooltip(barTimer, Loc['Adds a timer in seconds above the Casting Bar.']);
+    barTimer:SetText(Loc['Cast Bar Timer']);
+    AddTooltip(barTimer, Loc['Adds a timer in seconds above the Cast Bar.']);
 
     local castingScale = self:CreateSlider('castingScale');
     castingScale:SetPoint('TOPLEFT', barTimer, 'BOTTOMLEFT', 4, 0);
     castingScale:SetRange(0.1, 2.0);
     castingScale:SetStep(0.1);
-    AddTooltip(castingScale, Loc['Casting Bar Scale']);
+    AddTooltip(castingScale, Loc['Cast Bar Scale']);
```

Comment:
Same as before. Will need updating in `localisation\template.lua` so that any translators are using up to date texts.